build_tools/docker/cmake-python/Dockerfile

```diff
@@ -14,6 +14,8 @@ RUN apt-get update \
     python3-pip \
     python3-setuptools \
   && python3 -m pip install --upgrade pip \
-  && python3 -m pip install numpy==1.19.4 absl-py requests
+  && python3 -m pip install \
+    numpy==1.19.4 absl-py==0.12.0 requests PyYAML==5.4.1 wheel==0.36.2 \
+    pybind11==2.6.1
```

Comment:
Manual is fine for now, but we might also be able to hook in the requirements.txt file

Reply:
Yeah, I had the same thought and decided to do that "next time" :)